### PR TITLE
chore(deps): update paragon and frontend-build to openedx scope

### PR DESCRIPTION
**Pull request by brian-smith-tcril**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,15 +64,22 @@
...
+    "postcss-loader": "^8.0.0",
     "prettier": "2.8.1",
-    "rosie": "2.1.0"
+    "react-intl": "^6.6.1",
+    "rosie": "2.1.0",
+    "sass": "^1.70.0",
+    "sass-loader": "^14.0.0",
+    "sharp": "^0.33.2",
+    "source-map-loader": "^5.0.0",
+    "style-loader": "^3.3.4"
```
Comment (brian-smith-tcril) [inline package.json:83]: It's unclear to me why these aren't being found anymore when installed as part of frontend-build I'll continue to investigate.

File: package.json [hunk 2]
```diff
@@ -64,15 +64,18 @@
...
     "axios-mock-adapter": "1.21.2",
     "glob": "7.2.3",
     "husky": "7.0.4",
     "jest": "27.5.1",
+    "postcss-loader": "^8.0.0",
     "prettier": "2.8.1",
-    "rosie": "2.1.0"
+    "rosie": "2.1.0",
+    "sass-loader": "^14.0.0",
+    "source-map-loader": "^5.0.0"
```
Comment (brian-smith-tcril) [inline package.json:79]: I've narrowed it down to these 3

`postcss-loader`, `sass-loader`, and `source-map-loader`

`npm start` breaks without `postcss-loader` or `sass-loader`, but works without `source-map-loader`
`npm run build` breaks without `source-map-loader`

It's still not clear to me where the problem is living in the webpack config chain

Comment (brian-smith-tcril) [reply package.json]: resolved by doing a fresh regen of `package-lock.json` (thanks @arbrandes!)

**Review by arbrandes — APPROVED** [review]
+1

**Pull request merged**